docs/api/csf/csf-factories.mdx

```diff
@@ -0,0 +1,328 @@
+
+#### 4.1 Reusing story properties
+
+Previously, story properties such as `Story.args` or `Story.parameters` were accessed directly when reusing them in another story. While accessing them like this is still supported, it is deprecated in CSF Factories.
+
+All of the story properties are now contained within a new property called `composed` and should be accessed from that property instead. For instance, `Story.composed.args` or `Story.composed.parameters`.
+
+```diff title="Button.stories.ts"
+// ...rest of file
+
++ export const Primary = meta.story({
+- export const Primary: Story = {
+    args: { primary: true },
++ });
+- };
+
++ export const PrimaryDisabled = meta.story({
+- export const PrimaryDisabled: Story = {
+    args: {
++     ...Primary.composed.args,
+-     ...Primary.args,
+      disabled: true,
+    }
++ });
+- };
+```
+
+<Callout variant="info">
+  The property name "composed" was chosen because the values within are composed from the story, its component meta, and the preview configuration.
+
+  If you wish instead to access only the direct input to the story, you can use `Story.input` instead of `Story.composed`.
+
+  When running the [codemod](#codemod), it will automatically update usages of `Story.args` to `Story.input.args`. This is to ensure that your stories continue to work as expected. However, using `Story.composed.args` is likely more helpful in most cases.
+</Callout>
+
+### 5. Reusing stories in test files
+
+[Storybook's Test addon](../../writing-tests/test-addon.mdx) allows you to test your components directly inside Storybook. All of the stories are automatically turned into Vitest tests, so the integration is seamless in your testing suite.
+
+If you cannot use Storybook Test, you can still reuse the stories in your test files using [portable stories](../portable-stories/portable-stories-vitest.mdx). In prior story formats, you had to compose the stories before rendering them in your test files. With CSF Factories, you can now reuse the stories directly.
+
+```diff title="Button.test.ts"
+import { test, expect } from 'vitest';
+import { screen } from '@testing-library/react';
+- import { composeStories } from '@storybook/react';
+ 
+// Import all stories from the stories file
+import * as stories from './Button.stories';
+ 
++ const { Primary } = stories;
+- const { Primary } = composeStories(stories);
+ 
+test('renders primary button with default args', async () => {
+  // The run function will mount the component and run all of Storybook's lifecycle hooks
+  await Primary.run();
+  const buttonElement = screen.getByText('Text coming from args in stories file!');
+  expect(buttonElement).not.toBeNull();
+});
+```
+
+The `Story` object also provides a `Component` property in case you want to render it using another method of your choosing, for instance, with [Testing Library](https://testing-library.com/). You also have access to its composed properties (args, parameters, etc.) via the `composed` property.
```

Comment:
Words are not precious but we could go with something like this for clarity:
```suggestion
The `Story` object also provides a `Component` property, enabling you to render the component with any method you choose, such as [Testing Library](https://testing-library.com/). You can also access its composed properties (`args`, `parameters`, etc.) via the `composed` property.
```

Also, again a reminder to provide some links to the documentation when the rest of the snippets are added